### PR TITLE
Minor documentation fix for Pot Action handling

### PR DESCRIPTION
**Pull request by toddburnside**

### CODE REVIEW

File: doc/advanced/PotActions.md [hunk 1]
```diff
@@ -36,10 +36,11 @@ override def handle = {
       case PotPending =>
         noChange
       case PotReady =>
-        updated(action.value)
+        updated(action.potResult)
       case PotUnavailable =>
         updated(value.unavailable())
       case PotFailed =>
+        val ex = action.result.failed.get
```
Comment (ochrons) [inline doc/advanced/PotActions.md]: This should be `case PotFailed(ex)`

Comment (toddburnside) [inline doc/advanced/PotActions.md]: `PotFailed` is a case object and doesn't have any parameters. (The `Failed` case class, which extends Pot[Nothing], does have a `Throwable` parameter.) 

When I was trying to get my code to work, I based the above changes on the code in `PotAction.handler`.

Comment (ochrons) [inline doc/advanced/PotActions.md]: Ah, you're right of course!

**Comment by toddburnside** [comment]
BTW, I wanted to thank you not only for the great library, but also the extensive documentation for it. And all the other stuff you do for the community.

**Pull request merged**